oxo.py

```diff
@@ -1,26 +1,105 @@
 class OxoBoard:
     def __init__(self):
+    # def __init__(self, size)   <-- 1 of 3 to making the code take an size input from play.py
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.size = 3
+        # self.size = size    <-- 2 of 3 for variable size input
+        self.board = [[0 for x in range(self.size)] for y in range(self.size)]
+        '''
+         x = 1,2,3
+           [[a,b,c], y = 0
+            [d,e,f], y = 1
+            [g,h,i]] y = 2      i think
+            
+            it'd be pretty easy to implement bigger boards for this, need:
+            -> a screen that asks players how big they want the board
+            -> this __init__ to have an extra parameter for board size
+                -> this parameter also determines grid_width/height in play.py,
+                 the screen size would stay the same, but the boxes are smaller
+            -> the new parameter would be assigned to self.size
+            -> searches for non-diagonal wins would have to be updated, as they
+                are currently hard coded because it was nice and easy :(
+                
+            pep-8 is lame so I didn't stick too strictly to it,
```

Comment:
In professional software development you'll be expected to adhere to other people's coding standards even if you don't like them yourself. Better to get into the habit now!